velox/exec/StreamingAggregation.cpp

```diff
@@ -256,7 +279,14 @@ void StreamingAggregation::evaluateAggregates() {
     }
 
     if (isRawInput(step_)) {
-      function->addRawInput(inputGroups_.data(), rows, args, false);
+      if (function->clusteredInput() &&
```

Comment:
It is a bit strange to ask the function whether input is clustered. I believe this operator should know and the answer would be the same for all functions, no?

Reply:
Can we drop function->clusteredInput() check and rely solely on function->supportAddRawClusteredInput()?